### ISSUE TITLE
abnormal flagellar waveform

### ISSUE THREAD

**Issue by sbello** [obophenotype/mammalian-phenotype-ontology#3089]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
abnormal flagellar waveform

>2. Synonyms (e.g., absent spleen)
abnormal sperm beat pattern

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
Any anomaly in the waveform pattern created by the movement of the sperm tail

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

PMID:30759213
also 
J:148050 
PMID:24339785
PMID:26429887 
PMID:2346648 

Human syndrome reported in PMID:3688478 but the article is in German
